libs/gi/ui/src/components/artifact/ArtifactCard.tsx

```diff
@@ -529,12 +544,21 @@ function SubstatDisplay({
     <Box display="flex" gap={1} alignContent="center">
       <Typography
         sx={{ flexGrow: 1 }}
-        color={numRolls ? `${rollColor}.main` : 'error.main'}
+        color={
+          numRolls
+            ? `${rollColor}.main`
+            : !isActiveStat
+              ? 'secondary'
+              : 'error.main'
+        }
         component="span"
       >
         <StatIcon statKey={stat.key} iconProps={iconInlineProps} />{' '}
         {tk(stat.key)}
         {`+${artDisplayValue(stat.value, getUnitStr(stat.key))}${unit}`}
+        <Typography sx={{ ml: 0.5 }} component="span">
+          {!isActiveStat && '(Not Active)'}
```

Comment:
Should be a translateable string using t function. also can you make the english text be '(Unactivated)' to match in-game